client/gutenberg/extensions/simple-payments/edit.js

```diff
@@ -472,6 +472,7 @@ const mapSelectToProps = withSelect( ( select, props ) => {
 		: undefined;
 
 	return {
+		hasPublishAction: get( getCurrentPost(), [ '_links', 'wp:action-publish' ], false ),
```

Comment:
Let's get this to be consistently a boolean:
```suggestion
		hasPublishAction: !! get( getCurrentPost(), [ '_links', 'wp:action-publish' ] ),
```